src/js/components/aboutTheData/AgencyDetailsPage.jsx

```diff
@@ -76,80 +76,80 @@ const AgencyDetailsPage = () => {
-                        <ShareIcon slug={`submission-statistics/agency/${agencyCode}`} email={getAgencyDetailEmail(agencyOverview.name, agencyCode)} />
-                    </div>
-                )}
-            </StickyHeader>
-            <main id="main-content" className="main-content">
-                {loading && <LoadingMessage />}
-                {error && <ErrorMessage description={errorMessage} />}
-                {(!loading && !error) && (
-                    <>
-                        <div className="heading-container">
-                            <div className="back-link">
-                                <Link to={{
-                                    pathname: "/submission-statistics/",
-                                    search: `?${new URLSearchParams({ tab: 'submissions' }).toString()}`
-                                }}>
-                                    <FontAwesomeIcon icon="angle-left" />&nbsp;Back to All Agencies
-                                </Link>
-                            </div>
-                            <h2 className="header">{agencyOverview.name}</h2>
-                            <div className="agency-info">
-                                {agencyOverview.website && (
-                                    <div className="agency-info__group">
-                                        <h5>Agency Contact Information</h5>
-                                        <div className="more-info-note">Contact this Agency with questions about their submissions</div>
-                                        <div className="agency-info__website">
-                                            <ExternalLink url={agencyOverview.website} />
+            <PageHeader
+                title={agencyOverview?.name}
+                overLine="Agency Submission Data"
+                shareProps={{
+                    url: getBaseUrl(`submission-statistics/agency/${agencyCode}`),
+                    onShareOptionClick: handleShare
+                }}>
+                <main id="main-content" className="main-content">
+                    {loading && <LoadingMessage />}
+                    {error && <ErrorMessage description={errorMessage} />}
+                    {(!loading && !error) && (
+                        <>
+                            <div className="heading-container">
+                                <div className="back-link">
+                                    <Link to={{
+                                        pathname: "/submission-statistics/",
+                                        search: `?${new URLSearchParams({ tab: 'submissions' }).toString()}`
+                                    }}>
+                                        <FontAwesomeIcon icon="angle-left" />&nbsp;Back to All Agencies
+                                    </Link>
+                                </div>
+                                <h2 className="header">{agencyOverview.name}</h2>
```

Comment:
```suggestion
                                <h2 className="header">{agencyOverview?.name}</h2>
```
This page is breaking, I think because we need optional chaining here
```
Uncaught (in promise) TypeError: Cannot read property 'name' of null
    at AgencyDetailsPage (AgencyDetailsPage.jsx:207)
```